docs/source/recipes/server-side-rendering.md

```diff
@@ -113,7 +113,7 @@ class AppModule {
 
   onServer() {
     this.transferState.onSerialize(STATE_KEY, () =>
-      reurn this.cache.extract()
+      return this.cache.extract()
```

Comment:
shouldn't these be wrapped in braces to use return?

Reply:
I realized that there is always a better way to do this kind of things.
I'll try to remember the next time ;-)

Reply:
I mean, it's a SyntaxError, not that it's a better way :)

![screenshot 2017-11-28 09 54 32](https://user-images.githubusercontent.com/1162531/33310511-27ec816c-d422-11e7-80b6-c02db7601fcd.png)